pmm/pmm2-api-tests.groovy

```diff
@@ -73,72 +69,63 @@ pipeline {
                 sh 'git checkout ' + env.GIT_COMMIT_HASH
             }
         }
-        stage('Start staging') {
-            steps {
+        stage('API Tests Setup')
+        {
+            steps{
+                sh '''
+                    sudo curl -L https://github.com/docker/compose/releases/download/1.29.0/docker-compose-`uname -s`-`uname -m` | sudo tee /usr/local/bin/docker-compose > /dev/null
+                    sudo chmod +x /usr/local/bin/docker-compose
+                    sudo ln -sfn /usr/local/bin/docker-compose /usr/bin/docker-compose
+                    sudo docker-compose --version
+                    sudo docker build -t pmm-api-tests .
```

Comment:
We should be able to run these without `sudo`.

Reply:
I believe I tired, it was not working without sudo

Reply:
I tried, but was not working, will see thanks

Reply:
Actually the issue was with sh session, after allowing the group I needed to reconnect to sh for those permissions changes to reflect, fixed this in the latest commit.